x/conflict/README.md

```diff
@@ -9,9 +9,9 @@ The purpose of the conflict module is to ensure data reliability from providers
 
 ## Contents
 * [Concepts](#concepts)
-    * [Finalization Conflict](#finalization-conflict)
-    * [Self Provider Conflict](#self-provider-conflict)
     * [Response Conflict](#response-conflict)
+	* [Finalization Conflict](#finalization-conflict)
```

Comment:
align it to the other items in the list.
If it's supposed to be sub chapter of Response conflict, add '#' to the Finalization Conflict title